cmake/public/ComputeLibrary.cmake

```diff
@@ -0,0 +1,41 @@
+# Build with Compute Library backend for the Arm architecture
+# Note: Compute Library is available from: https://github.com/ARM-software/ComputeLibrary
+#   and must be built separately. The location of the Compute Library build
+#   must be set with the env var ACL_ROOT_DIR. This path will be checked later
+#   as part of FindACL.cmake in oneDNN.
+
+IF(NOT CPU_AARCH64)
+    RETURN()
+ENDIF()
+
+if(NOT USE_MKLDNN_ACL)
+    RETURN()
+ENDIF()
+
+SET(DNNL_AARCH64_USE_ACL ON CACHE BOOL "" FORCE)
+
+# Check the Compute Library version number.
+# Note: oneDNN / MKL-DNN v2.2 onwards will check the Compute Library version
+#   the version check here can be removed once PyTorch transitions to v2.2.
+SET(ACL_MINIMUM_VERSION "21.02")
+
+FILE(GLOB_RECURSE ACL_VERSION_FILE $ENV{ACL_ROOT_DIR}/*/arm_compute_version.embed)
+
+IF("${ACL_VERSION_FILE}" STREQUAL "")
+  MESSAGE(WARNING "Build may fail: Could not determine ACL version (minimum required is ${ACL_MINIMUM_VERSION})")
+ELSE()
+  FILE(READ ${ACL_VERSION_FILE} ACL_VERSION_STRING)
+  STRING(REGEX MATCH "v([0-9]+\\.[0-9]+)" ACL_VERSION ${ACL_VERSION_STRING})
+  SET(ACL_VERSION "${CMAKE_MATCH_1}")
+
+  IF(${ACL_VERSION} VERSION_EQUAL "0.0")
+    # Unreleased ACL versions come with version string "v0.0-unreleased", and may not be compatible with oneDNN.
+    # It is recommended to use the latest release of ACL.
+    MESSAGE(WARNING "Build may fail: Using unreleased ACL version (minimum required is ${ACL_MINIMUM_VERSION})")
+  ELSEIF(${ACL_VERSION} VERSION_LESS ${ACL_MINIMUM_VERSION})
+    MESSAGE(FATAL_ERROR "Detected ACL version ${ACL_VERSION}, but minimum required is ${ACL_MINIMUM_VERSION}")
+  ENDIF()
+ENDIF()
+
+SET(CMAKE_CXX_STANDARD 14)
```

Comment:
Please do not define C++ standard for the entire project here.

Reply:
As it turns out this isn't needed here - I've taken it out.